wasp/src/main/java/com/orhanobut/wasp/IOUtils.java

```diff
@@ -0,0 +1,37 @@
+package com.orhanobut.wasp;
+
+import android.content.Context;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+/**
+ * @author Emmar Kardeslik
+ */
+public class IOUtils {
+
+    private IOUtils() {
+        //no instance
+    }
+
+    public static String readFileFromAssets(final Context context, final String filePath) throws IOException {
```

Comment:
I think we don't need final declaration for Context and filePath here. For the consistency I would not use them.